doc/Installation/Installation-Ubuntu-1604-Apache.md

```diff
@@ -98,9 +98,17 @@ Edit the text which says `RANDOMSTRINGGOESHERE` and set your own community strin
     chmod +x /usr/bin/distro
     systemctl restart snmpd
 
-### Cron job
+### Background jobs
 
-    cp /opt/librenms/librenms.nonroot.cron /etc/cron.d/librenms
+If your upgrading LibreNMS, ensure to remove the crontab:
+
+    rm -fv /etc/cron.d/librenms
+
+Copy, enable and start systemd timers:
+
+    cp /opt/librenms/contrib/systemd/* /lib/systemd/system/
```

Comment:
Custom unit files should not be copied to /lib/systemd/system, they should be copied to /etc/systemd/system.

Reply:
Fixed